zul/src/main/resources/web/js/zul/inp/Doublespinner.ts

```diff
@@ -116,7 +116,7 @@ export class Doublespinner extends zul.inp.NumberInputWidget<number> {
 
 	override setConstraint(constraint: string): this {
 		if (typeof constraint == 'string' && !constraint.startsWith('[')/*by server*/) {
-			var simpleDoubleSpinnerConstraint = new zul.inp.SimpleDoubleSpinnerConstraint(constraint);
+			var simpleDoubleSpinnerConstraint = new zul.inp.SimpleNumberInputConstraint(constraint);
```

Comment:
for backward compatibility, we cannot change the class name directly in a minor release version.